templates/registration/logged_out_.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}

```

Comment:
The file name `logged_out_.html` contains an underscore at the end. Ensure this is intentional, as Django's default template for logout is `logged_out.html`. If this is not intentional, consider renaming the file to `logged_out.html` to follow Django's conventions.